test/behavior.t/run.t

```diff
@@ -18,15 +18,15 @@
-  {"sample_id":9,"cmd":"ocamlmerlin server occurrences -identifier-at '3:12' -filename test1.ml < test1.ml","responses":[{"class":"return","value":[{"start":{"line":3,"col":12},"end":{"line":3,"col":13}}],"notifications":[],"query_num":9}]}
-  {"sample_id":8,"cmd":"ocamlmerlin server type-enclosing -position '1:8' -index 0 -filename test1.ml < test1.ml","responses":[{"class":"return","value":[{"start":{"line":1,"col":8},"end":{"line":1,"col":9},"type":"int","tail":"no"}],"notifications":[],"query_num":8}]}
-  {"sample_id":7,"cmd":"ocamlmerlin server case-analysis -start '1:8' -end '1:8' -filename test1.ml < test1.ml","responses":[{"class":"return","value":[{"start":{"line":1,"col":8},"end":{"line":1,"col":9}},"match 1 with | 0 -> _ | _ -> _"],"notifications":[],"query_num":7}]}
-  {"sample_id":5,"cmd":" ocamlmerlin server locate -look-for ml -position '3:12' -filename test.ml < test.ml","responses":[{"class":"return","value":{"file":"stdlib.ml","pos":{"line":94,"col":0}},"notifications":[],"query_num":5}]}
-  {"sample_id":4,"cmd":"ocamlmerlin server expand-prefix -prefix '( +' -position '3:12' -filename test.ml < test.ml","responses":[{"class":"return","value":{"entries":[{"name":"()","kind":"Constructor","desc":"","info":"","deprecated":false}],"context":null},"notifications":[],"query_num":4}]}
-  {"sample_id":3,"cmd":"ocamlmerlin server complete-prefix -prefix '( +' -position '3:12' -filename test.ml < test.ml","responses":[{"class":"return","value":{"entries":[],"context":["application",{"argument_type":"'a","labels":[]}]},"notifications":[],"query_num":3}]}
-  {"sample_id":2,"cmd":"ocamlmerlin server occurrences -identifier-at '3:12' -filename test.ml < test.ml","responses":[{"class":"return","value":[{"start":{"line":3,"col":12},"end":{"line":3,"col":13}}],"notifications":[],"query_num":2}]}
-  {"sample_id":1,"cmd":"ocamlmerlin server type-enclosing -position '3:12' -index 0 -filename test.ml < test.ml","responses":[{"class":"return","value":[{"start":{"line":3,"col":12},"end":{"line":3,"col":13},"type":"int -> int -> int","tail":"no"},{"start":{"line":3,"col":10},"end":{"line":3,"col":15},"type":1,"tail":"no"},{"start":{"line":3,"col":6},"end":{"line":3,"col":15},"type":2,"tail":"no"}],"notifications":[],"query_num":1}]}
-  {"sample_id":0,"cmd":"ocamlmerlin server case-analysis -start '3:14' -end '3:14' -filename test.ml < test.ml","responses":[{"class":"return","value":[{"start":{"line":3,"col":14},"end":{"line":3,"col":15}},"(match 3 with | 0 -> _ | _ -> _)"],"notifications":[],"query_num":0}]}
+  {"sample_id":12,"cmd":" ocamlmerlin server locate -look-for ml -position '3:12' -filename test1.ml < test1.ml","responses":[{"class":"return","value":{"file":"stdlib.ml","pos":{"line":94,"col":0}},"notifications":[],"heap_mbytes":4,"cache":{"reader_phase":"miss","ppx_phase":"miss","typer":{"reused":1,"typed":1},"cmt":{"hit":2,"miss":0},"cmi":{"hit":1,"miss":0}},"query_num":12}]}
```

Comment:
I'd filter `heap_mbytes` out, potentially also the cache hits and misses component. The `heap_mbytes`captures how much Merlin allocates during the query. That's not deterministic.

Reply:
Maybe I can leave it to the current PR i am currently working on?

Reply:
Sure! In that case, I'd remove the test file modifications in this PR.

(This repo isn't at all kept very clean, though, so I won't be stubborn on it. If you prefer to merge this with unclean tests and fix the tests later, let's just do that.)